tools/benchmark/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-tools/benchmark",
-  "version": "0.40.0",
+  "version": "0.41.0",
```

Comment:
Here is the version bump for publishing the new fluid-tools/benchmark: everything else in this update is just updating the versions of mocha and @types/moch used.